### PR TITLE
Add corona chart

### PR DESCRIPTION
**Pull request by flsobral**
add corona-chart sample
#3

### CODE REVIEW

File: corona-chart/src/main/java/com/totalcross/coronachart/RunCoronaChartApplication.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package com.totalcross.coronachart;
+
+import totalcross.TotalCrossApplication;
+
+public class RunCoronaChartApplication {
+    public static void main(String[] args) {
+        TotalCrossApplication.run(CoronaChartApp.class,
+                // "/scr", "1920x1280",
+                // "/scr", "960x640",
+                "/scr", "848x480", "/density", "1", "/r", "5443444B5AAEEB90306B00E4");
```
Comment (VaneskaSousa) [inline corona-chart/src/main/java/com/totalcross/coronachart/RunCoronaChartApplication.java:10]: Please delete the comments and the TC Key

**Review by VaneskaSousa — CHANGES_REQUESTED** [review]
Just need to delete the TC Key from RunCoronaChartApplication.java

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -13,6 +13,7 @@ jobs:
           matrix:
             projects:
               - HomeApplianceXML
+              - corona-chart
```
Comment (acmlira) [inline .github/workflows/ci.yml:16]: Why not _CoronaChart_?

**Review by VaneskaSousa — APPROVED** [review]

**Comment by nmarquesin** [comment]
@flsobral, could you please update the commit message to the correct format "<scope>: <subject>" with an optional body and footer separated by blank lines?

**Comment by VaneskaSousa** [comment]
Hey, @flsobral what are missing to merge this branch?

**Comment by flsobral** [comment]
it's hammer time

**Pull request closed without merging**